src/main/java/com/microsoft/sqlserver/jdbc/SQLServerBlob.java

```diff
@@ -53,6 +53,12 @@
 
     private final String traceID;
 
+    private boolean loadLob = false;
+
+    void setStreaming(boolean b) {
+        loadLob = b;
```

Comment:
Can you refactor this `loadLob` variable name to `delayLoadingLobs` to maintain consistency? It will make it easier to search for its usage in the future if we were to come back to this feature

Reply:
never mind, I realized that this variable is actually the opposite of `delayLoadingLobs`. I'd prefer if we had one type of boolean to represent the connection property (instead of having two booleans opposite of each other's values that are meant for the same purpose), but if others are fine with this, I am too.

Reply:
Overall I think this setStreaming naming is confusing. The connection property `delayLoadingLobs` is true by default, which means we want to stream by default. Setting `delayLoadingLobs` to false ends up calling this method setStreaming to true, which makes the driver *not* stream (by calling getStringFromStream method). Unless I'm mistaken about the functionalities, we should rename the methods / variables to add more clarity to what the driver is doing.